tools/convert_checkpoint/weights2megatron/weights2megatron_llama.py

```diff
@@ -0,0 +1,186 @@
+import os
```

Comment:
For consistency with others, this conversion script should be named "transformers_to_megatron*.py".